NBE1_1_9team-main/src/main/java/gc/cafe/api/controller/user/request/JoinRequest.java

```diff
@@ -0,0 +1,10 @@
+package gc.cafe.api.controller.user.request;
+
+import gc.cafe.api.service.user.request.JoinServiceRequest;
+
+public record JoinRequest(String email, String name, String password) {
```

Comment:
`@Valid`는 있는데 실제론 Validation을 안하는데 record에서는 그냥 파라미터에 `@NotBlank` 같이 붙여주면 되나요 ?
레코드를 제대로 이용안해봐서 궁금하네요

Reply:
아 validation 적용하려고 했는데 까먹었네여.. 아래처럼 할 수 있습니다! 
```suggestion
public record JoinRequest(

    @NotNull
    String email, 
    
    @NotNull
    String name, 
    
    @NotNull
    String password
) {
```